autoload/OmniSharp/util.vim

```diff
@@ -275,7 +275,23 @@ function! OmniSharp#util#TranslatePathForClient(filename) abort
   if filename =~# '\$metadata\$'
     let filename = OmniSharp#util#TempDir() . '/' . fnamemodify(filename, ':t')
   endif
-  return fnamemodify(filename, ':.')
+
+  let modifiers = get(g:, 'OmniSharp_filename_modifiers', ':.')
+
+  if modifiers ==# 'relative'
+    let common = escape(getcwd(), '\')
+    let result = ''
+    while substitute(filename, common . s:dir_separator, '', '') ==# filename
+    \ && common !=# s:dir_separator
```

Comment:
For windows this needs a different stop condition. Maybe rather
```vim
common ==# fnamemodify(common, ":h")
```
?

Reply:
Oh yep, I haven't tested on Windows 😊 . Thanks for the reminder.